modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-evaluate-availability_{context}"]
+= Evaluating update availability
+
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service for the most recent graph of update possibilities.
+This graph is based on the channel to which the cluster is subscribed.
+If the graph contains paths to OCP versions that are newer than the cluster's current version, the CVO saves these versions into the `available updates` field of its ClusterVersion resource.
+
+The CVO continuously tests the available updates for conditional update risks.
+These risks are conveyed through the data served by the OpenShift Update Service, which contains information for each version about known issues that may affect a cluster updated to that version.
+Most risks are limited to clusters with specific characteristics, such as clusters with a certain size or clusters that are deployed in a particular cloud platform.
+
+The CVO checks its cluster characteristics against the conditional risk information for each available update, and if it finds that the cluster matches the criteria, it also stores this information in its ClusterVersion resource.
+The user interface (web console and CLI) then presents this information to the administrator.
+The update recommendation contains a link to further resources about the risk so that the administrator can make an informed decision about the update.
```

Comment:
> The update recommendation contains a link

The "supported but not recommended" update recommendation contains a link ...

Reply:
"not recommended update recommendation" :smile: